docs/book/src/reference/glossary.md

```diff
@@ -191,6 +217,10 @@ A generically understood combination of a kernel and system-level userspace inte
 # P
 ---
 
+### Patch 
+
+A well-defined set of interfaces and APIs to describe modifications to a Kubernetes object.
```

Comment:
```suggestion
A set of instructions describing modifications to a Kubernetes object. Depending on the context it could be either a JSON Patch or a JSON Merge Patch.
```
wdyt?

Reply:
I'd prefer to leave this high level and not to go over and back too on the wording @vincepri WDYT?

Reply:
TBH I have some concerns given that I struggle to identify a patch as a "well-defined set of interfaces and APIs"